kore/src/main/scala/org/kframework/attributes/Att.scala

```diff
@@ -166,6 +166,7 @@ object Att {
   final val FUNCTION = Key("function", KeyType.BuiltIn)
   final val FUNCTIONAL = Key("functional", KeyType.BuiltIn)
   final val GROUP = Key("group", KeyType.BuiltIn)
+  final val HASKELL = Key("haskell", KeyType.BuiltIn)
```

Comment:
Is this being used anywhere as an attribute?

Reply:
It's used in `domains.md`, e.g., on `MAP-KORE-SYMBOLIC`.

Dwight requested that I whitelist it for use with the Maude backend - both the haskell backend and Maude backend are `symbolic`, so we need a separate attribute to distinguish modules only for use by the haskell backend.